src/main/core/worker.rs

```diff
@@ -346,12 +349,17 @@ impl Worker {
         let src_ip: std::net::Ipv4Addr = u32::from_be(src_ip).into();
         let dst_ip: std::net::Ipv4Addr = u32::from_be(dst_ip).into();
 
-        let dst_host_id = Worker::with(|w| {
-            w.shared
-                .resolve_ip_to_host_id(dst_ip)
-                .expect("No host ID for dest address {dst_ip}")
-        })
-        .unwrap();
+        let Some(dst_host_id) = Worker::with(|w| w.shared.resolve_ip_to_host_id(dst_ip)).unwrap()
+        else {
+            log_once_per_value_at_level!(
```

Comment:
Should this also set the delivery status to `cshadow::_PacketDeliveryStatusFlags_PDS_INET_DROPPED` as done for random packet drops below?

Reply:
Good catch! Added.